src/System.AppContext/ref/System.AppContext.csproj

```diff
@@ -1,6 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <CheckForOverflowUnderflow>true</CheckForOverflowUnderflow>
```

Comment:
So, I've experimented with this for a few hours now and you're right, there are tons of exceptions. I've started fixing them one by one but that'll take time. Though, rethinking this, I'd still prefer to turn the setting on globally rather than first adding it to project-by-project only to remove it later (there are about 180 project after all). Also, since there seems to be an agreement that explicitly marking code that needs to run unchecked as such is a good thing, how about this:
- We abandon this PR
- I'll start submitting a series of small(-ish) PR with unchecked added where needed
- Only the final PR will include the change to dir.props in the root so there should be little risk of regressions